web/styles/compose.css

```diff
@@ -642,12 +642,14 @@
     right: 0;
     color: var(--color-compose-send-control-button);
     background: transparent;
-    font-size: 12.5px;
+    /* 12.5px at 14px em */
+    font-size: 0.8928em;
```

Comment:
I wonder if 14px is the right em value here; I seem to recall this close button being introduced after the initial info density work at 16px, and it does look a little large to me in the 16px font size.